web/onos-gui/src/app/onos-topo/device-detail/device-detail.component.css

```diff
@@ -31,18 +31,22 @@
  */
 
 
-#device-detail-panel.floatpanel {
+ #device-detail-panel.floatpanel {
     z-index: 0;
     font-size: 10pt;
     top: 185px;
-    height: 400px;
-    overflow-y: scroll;
+    height: 50vh;
```

Comment:
the 50vh isn't really doing the job - maybe change it to 550px

we are trying to handle 2 variables here

1. a variable number of attributes
2. a variable screen height - I'm less worried about this - I thought it would be possible to solve it with "vh" but I don't think it is